components/section/section-error-boundary.tsx

```diff
@@ -0,0 +1,51 @@
+import { ErrorBoundary as SentryErrorBoundary } from '@sentry/nextjs';
+import Warning from '#components-ui/alerts/warning';
+import constants from '#models/constants';
+
+export default function SectionErrorBoundary({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <SentryErrorBoundary
+      fallback={(error) => (
+        <div>
+          {error instanceof SyntaxError && window.IS_OUTDATED_BROWSER ? (
+            <>
+              <Warning>
+                Votre navigateur est trop vieux pour afficher cette section.
+              </Warning>
+              <p>
+                Avoir un navigateur à jour est{' '}
+                <a
+                  target="_blank"
+                  rel="noreferrer noopener"
+                  href="https://www.ssi.gouv.fr/entreprise/precautions-elementaires/bonnes-pratiques-de-navigation-sur-linternet/"
+                >
+                  fortement recommandé par l’ANSSI
+                </a>{' '}
+                pour naviguer sur internet en sécurité.
+              </p>
+            </>
```

Comment:
I think this is overkill as if `window.IS_OUTDATED_BROWSER` is `true` then user should already have this message on top of the page